### PR TITLE
[docs] Updates to RN 2.18.5.0

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/releases/release-notes/v2.18/#v2.18.5.0

### CODE REVIEW

File: docs/content/preview/releases/release-notes/v2.18.md [hunk 1]
```diff
@@ -59,64 +59,117 @@ For an RSS feed of all release series, point your feed reader to the [RSS feed f
 docker pull yugabytedb/yugabyte:2.18.5.0-b93
 ```
 
-### New Features
+### New features
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.18.md:62]: These are both bug fixes

File: docs/content/preview/releases/release-notes/v2.18.md [hunk 2]
```diff
@@ -59,64 +59,117 @@ For an RSS feed of all release series, point your feed reader to the [RSS feed f
...
+* Upgrades OpenSSL to version 3.0.8 as the previous version reached End of Life, pulling in changes to disable Linuxbrew builds and update glog for stack unwinding based on the backtrace function. {{<issue 19736>}}
+* Ensures the `master_join_existing_universe` flag value check before restoring the initial sys catalog snapshot to avoid incorrect operations. {{<issue 19357>}}
+* Enables automatic recovery of index tables for bugs detected in the context of all new index tables created from 2.14; prevents retention of tombstones in SST files, mitigates performance degradation and disk size leak. {{<issue 19731>}}
+
 #### CDC
 
-* Introduces a flag `enable_tablet_split_of_cdcsdk_streamed_tables` (default:`false`) to toggle automatic tablet splitting for tables in a CDCSDK stream, enhancing user control over replication processes. {{<issue 19482>}}
+* Introduces a GFlag: `enable_tablet_split_of_cdcsdk_streamed_tables` (default:`false`) to toggle automatic tablet splitting for tables within a CDCSDK stream, enhancing user control over replication processes. {{<issue 19482>}}
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.18.md:108]: new feature maybe

File: docs/content/preview/releases/release-notes/v2.18.md [hunk 3]
```diff
@@ -59,64 +59,117 @@ For an RSS feed of all release series, point your feed reader to the [RSS feed f
...
+* Rectifies gcc11 compilation failure on CentOS by modifying iterator to non-const. {{<issue 20087>}}
+* Enhances debugging capabilities in YSQL Webserver process by introducing two new flags for endpoint access logging and tcmalloc stats printing. Also, values can now be updated via a SIGHUP signal, with `yb_pg_metrics` enabled to handle this signal. {{<issue 20157>}}
+* Resolves a segmentation fault issue in postmaster by enabling it to use the Proc struct of a killed process, thereby enhancing process cleanup during unexpected terminations. {{<issue 20166>}}
+* Restart the postmaster whenever a process is killed while writing in a critical section, preventing corrupted state and CPU consumption due to infinite loops. {{<issue 20255>}}
+* Enhances YSQL webserver with new `/memz` and `/webserver-heap-prof` endpoints for greater visibility into webserver memory usage. Also introduces a new runtime variable `yb_pg_metrics.webserver_profiler_sample_freq_bytes` for controlling tcmalloc sampling frequency. {{<issue 20157>}}
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.18.md:136]: improvement

File: docs/content/preview/releases/release-notes/v2.18.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/releases/release-notes/v2.18.md:62]: No 19815 and 19746 are marked as new features in GitHub

File: docs/content/preview/releases/release-notes/v2.18.md [hunk 4]
```diff
@@ -59,64 +59,117 @@ For an RSS feed of all release series, point your feed reader to the [RSS feed f
...
+* Resolves data inconsistency between source and target instances during Xcluster +Transactions +Aggressive tablet Splitting operations. Now, ensures accurate application of intents matching the producer tablet's key range on the consumer, facilitating correct data replication in diverse transaction scenarios. {{<issue 19728>}}
 
 #### CDC
 
 * Allows continuation of tablet fetching, even if certain tables face errors, by logging a warning instead of sending unnecessary errors to the client. {{<issue 19434>}}
 * Returns error to the client if reading the `cdc_state` fails, ensuring consistent and correct list of tablets, enabling your client to take suitable action. {{<issue 19428>}}
 * Eliminates deadlocks during database drops by properly handling namespace-level CDC stream deletion and setting up the database deletion process to finish in a finite time. {{<issue 19879>}}
 
-#### DocDB
+### Other
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.18.md:166]: these look like bug fixes, make it an h4

**Review by ddhodge — APPROVED** [review]
few comments

File: docs/content/preview/releases/release-notes/v2.18.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/releases/release-notes/v2.18.md:62]: moved to bugs

**Review by premkumr — APPROVED** [review]
LGTM

**Pull request merged**